cmd/issue_create.go

```diff
@@ -138,6 +143,7 @@ func init() {
 	issueCreateCmd.Flags().StringSliceP("label", "l", []string{}, "Set the given label(s) on the created issue")
 	issueCreateCmd.Flags().StringSliceP("assignees", "a", []string{}, "Set assignees by username")
 	issueCreateCmd.Flags().StringP("template", "t", "default", "use the given issue template")
+	issueCreateCmd.Flags().Bool("rawtext", false, "Do not use markdown renderer for text")
```

Comment:
`force-linebreak` comes to mind

Reply:
or even `--br` named after the html for it.

Reply:
The help text here should capture what this feature is doing, less what it's trying to undo, which hopefully will make naming the feature and writing a description easier.

`"append 2 spaces to the end of each line to force markdown linebreaks"` <- even an off the cuff description makes it pretty clear what using the flag is going to do. Just my 0.002 cents